config/redis.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const config = require('config');
+
+const redisConfig = config.get('redis');
+
+const queuePrefix = redisConfig.prefix || '';
+
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: redisConfig.host,
+    options: {
+        password: redisConfig.password
+    },
+    port: redisConfig.port,
+    database: 0
+};
+
+module.exports = {
```

Comment:
All modules not directly tested should be mocked out. That's how it is with all our unit tests.